assets/src/dashboard/app/api/wpAdapter.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const get = (path, options = {}) => apiFetch({ path, ...options });
+
+const post = (path, options = {}) =>
+  apiFetch({
+    path,
+    ...options,
+    method: 'POST',
+  });
+
+const deleteRequest = (path, options = {}) =>
+  apiFetch({
+    path,
+    ...options,
+    method: 'DELETE',
+  });
+
+export default {
+  get,
+  post,
+  deleteRequest,
+};
```

Comment:
This is a simple wrapper around our wordpress api.  The idea is that we can swap in a new adapter with the same interface and our api code will continue working.